app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -116,6 +117,51 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       }
     },
 
+    populateLinkPathParts: function (clickData) {
+      var href = clickData.url
+      var path = ''
+      if (this.hrefIsRelative(href)) {
+        path = href
+      } else if (this.isMailToLink(href)) {
+        path = href
+      } else {
+        path = href.replace(/^(http:||https:)?(\/\/)([^\/]*)/, '') // eslint-disable-line no-useless-escape
+      }
+
+      if (path === '/' || path.length === 0) {
+        return
+      }
+
+      /* Divide the length into hundreds, to determine how many string "parts" to make.
+      For example a 150 char string length divided by 100 would return 1.5,
+      so we round up, which returns 2, which makes sense as we would need one segment for chars 0-100, and another for chars 100-150 . */
+      var hundreds = Math.ceil(path.length / 100)
+
+      var LIMIT = 5
+
+      if (hundreds > LIMIT) {
```

Comment:
Not sure which would be more efficient, but you can also do `hundreds = Math.min(hundreds, LIMIT)`

Reply:
Good idea :+1: